templates/nodeovc/schema.capnp

```diff
@@ -0,0 +1,76 @@
+@0xd37ff48ad935931f;
+struct Schema {
+	# Description for the VM
+	description @0 :Text;
+
+	# Name of the VM
+	name @1 :Text;
```

Comment:
no need to be there I think, if you can use the name of the service itself, this is redundant info